client/src/modules/events/pages/eventPage.tsx

```diff
@@ -353,72 +354,77 @@ export const EventPage: NextPage = () => {
         <Text fontWeight={'500'} fontSize={['smaller', 'sm', 'md']}>
           Ending: {endsAt}
         </Text>
-        {!attendanceStatus || attendanceStatus === 'no' ? (
-          <Button
-            colorScheme="blue"
-            data-cy="attend-button"
-            isLoading={loadingAttend}
-            onClick={() => tryToAttend()}
-            paddingBlock="1"
-            paddingInline="2"
-          >
-            {data.event.invite_only ? 'Request' : 'Attend'}
-          </Button>
-        ) : (
-          <HStack>
-            {attendanceStatus === 'waitlist' ? (
-              <Text fontSize="md" fontWeight="500">
-                {data.event.invite_only
-                  ? 'Event owner will soon confirm your request'
-                  : "You're on waitlist for this event"}
-              </Text>
-            ) : (
-              <Text data-cy="attend-success">You are attending this event</Text>
-            )}
+        <SimpleGrid columns={2} gap={5} alignItems="center">
+          {!attendanceStatus || attendanceStatus === 'no' ? (
             <Button
-              isLoading={loadingCancel}
-              onClick={onCancelAttendance}
+              colorScheme="blue"
+              data-cy="attend-button"
+              gridColumnStart="span 2"
+              isLoading={loadingAttend}
+              onClick={() => tryToAttend()}
               paddingBlock="1"
               paddingInline="2"
             >
-              Cancel
+              {data.event.invite_only ? 'Request Invite' : 'Attend Event'}
             </Button>
-          </HStack>
-        )}
-        {userEvent && (
-          <HStack>
-            {userEvent.subscribed ? (
-              <>
-                <Text fontSize={'md'} fontWeight={'500'}>
-                  You are subscribed to event updates
+          ) : (
+            <>
+              {attendanceStatus === 'waitlist' ? (
+                <Text fontSize="md" fontWeight="500">
```

Comment:
these styles are different from the other text, which weirding me out.

but this isn't related to the original issue, I am happy with current changes, if you think this deserves to be in another PR.

```suggestion
                <Text>
```

Reply:
I suppose there's couple more inconsistencies between the text - ie. text by subscribe/unsubscribe buttons have such styling, so I'd say separate PR would be better to get them all together. And to figure out whether the styling is needed or not.